Dockerfile

```diff
@@ -1,26 +1,31 @@
+# vim:set ft=dockerfile:
 FROM pcic/geospatial-python:gdal3
-
 MAINTAINER https://github.com/pacificclimate/sandpiper
 LABEL Description="sandpiper WPS" Vendor="pacificclimate" Version="0.1.0"
 
 ENV PIP_INDEX_URL="https://pypi.pacificclimate.org/simple/"
 ENV THREDDS_URL_ROOT="https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets"
 
-WORKDIR /code
+# Update system
+RUN apk add \
+  libxml2-dev \
+  libxslt-dev \
+  linux-headers
 
-COPY requirements.txt ./
+COPY . /opt/wps
 
-RUN apk add libxml2-dev libxslt-dev linux-headers && \
-    pip3 install psutil && \
-    pip3 install sphinx && \
-    pip3 install -r requirements.txt --ignore-installed && \
-    pip3 install gunicorn
+WORKDIR /opt/wps
```

Comment:
Does changing the working directory from `code` to `/opt/wps` make any difference?

Reply:
Not really, this is just to match what `bird-house` does.